specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.5/TextAnalytics.json

```diff
@@ -0,0 +1,2931 @@
+            "notStarted",
+            "running",
+            "succeeded",
+            "failed",
+            "cancelled",
+            "cancelling",
+            "partiallyCompleted"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": false,
+            "name": "State"
+          }
+        }
+      },
+      "required": [
+        "jobId",
+        "lastUpdateDateTime",
+        "createdDateTime",
+        "status"
+      ],
+      "type": "object"
+    },
+    "AnalyzeJobMetadata": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/JobMetadata"
+        },
+        {
+          "properties": {
+            "displayName": {
+              "type": "string"
+            }
+          }
+        }
+      ]
+    },
+    "KeyPhraseResult": {
+      "type": "object",
+      "required": [
+        "documents",
+        "errors",
+        "modelVersion"
+      ],
+      "properties": {
+        "documents": {
+          "type": "array",
+          "description": "Response by document",
+          "items": {
+            "$ref": "#/definitions/DocumentKeyPhrases"
+          }
+        },
+        "errors": {
+          "type": "array",
+          "description": "Errors by document id.",
+          "items": {
+            "$ref": "#/definitions/DocumentError"
+          }
+        },
+        "statistics": {
+          "$ref": "#/definitions/RequestStatistics"
+        },
+        "modelVersion": {
+          "type": "string",
+          "description": "This field indicates which model is used for scoring."
+        }
+      }
+    },
+    "DocumentKeyPhrases": {
+      "type": "object",
+      "required": [
+        "id",
+        "keyPhrases",
+        "warnings"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Unique, non-empty document identifier."
+        },
+        "keyPhrases": {
+          "type": "array",
+          "description": "A list of representative words or phrases. The number of key phrases returned is proportional to the number of words in the input document.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "warnings": {
+          "type": "array",
+          "description": "Warnings encountered while processing document.",
+          "items": {
+            "$ref": "#/definitions/TextAnalyticsWarning"
+          }
+        },
+        "statistics": {
+          "description": "if showStats=true was specified in the request this field will contain information about the document payload.",
+          "$ref": "#/definitions/DocumentStatistics"
+        }
+      }
+    },
+    "KeyPhrasesTask": {
+      "properties": {
+        "parameters": {
+          "properties": {
+            "model-version": {
+              "default": "latest",
+              "type": "string"
+            },
+            "loggingOptOut": {
+              "default": false,
```

Comment:
```suggestion
              "default": "false",
```

Reply:
@deyaaeldeen This change actually breaks the python generation. Wondering if the service should be following how opinionMining is defined here: https://github.com/Azure/azure-rest-api-specs/blob/1397ebcd89b83e94c674db1763b5fe97c45e74e3/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.5/TextAnalytics.json#L945-L947

Reply:
@kristapratico interesting, it is not clear to me which way is the right syntax, perhaps it is both? @isabelmsft do you know?

Reply:
@iscai-msft

Reply:
It should be what it originally was, a boolean value of `false`. This is because we listed it as a boolean on line 943. @deyaaeldeen does having it as a boolean value break you?

Reply:
@laramume please switch boolean values to be without quotes. This breaks the JS code gen but I will work on fixing it. cc @joheredi

Reply:
ohh interesting. How is `opinionmining` working for JS? I think the swagger has it as boolean

Reply:
Changed it. Can someone merge it as I dont have permission for it